### PR TITLE
Preserve timestamps on binary copy.

### PR DESCRIPTION
**Pull request by thmsbinder**

### CODE REVIEW

File: kokoro/build_binaries_oak_containers.sh [hunk 1]
```diff
@@ -18,21 +18,26 @@ cd "$(dirname "$0")/.."
 ./scripts/docker_run nix develop .#containers --command just kokoro_oak_containers
 
 mkdir -p "$KOKORO_ARTIFACTS_DIR/test_logs/"
-cp ./target/nextest/default/*.xml "$KOKORO_ARTIFACTS_DIR/test_logs/" || true
+cp --preserve=timestamps \
```
Comment (rbehjati) [inline kokoro/build_binaries_oak_containers.sh:21]: Does the timestamp matter for `test_logs`?

**Review by rbehjati — APPROVED** [review]
Nice. Thanks.

File: kokoro/build_binaries_oak_containers.sh (hunk 1 above)
Comment (thmsbinder) [reply kokoro/build_binaries_oak_containers.sh:21]: No, makes no difference. It is just more consistent. The log file is closed and will continue to show the last time anything was written to it.

**Review by andrisaar — APPROVED** [review]
Can you elaborate as to why this is needed?

**Comment by thmsbinder** [comment]
> Can you elaborate as to why this is needed?

We use the mtime value of the binary file in the `created` timestamp of the binary metadata proto, for any binary package. Preserving the timestamps will make this value a little more accurate (earlier) henceforth (even though it was already in the right ballpark).

On a higher level, motivation was to provide means to sort binaries by the order in which they were produced. We don't retain the pull request number or the Kokoro run number. Comparing creation timestamps should be suitable.

**Pull request merged**